src/reports/package/footer-text-for-service.tsx

```diff
@@ -2,18 +2,28 @@
 // Licensed under the MIT License.
 import * as React from 'react';
 
+import { EnvironmentInfoProvider } from 'common/environment-info-provider';
 import { ReportSectionFactory } from 'reports/components/report-sections/report-section-factory';
 import { ToolLink } from 'reports/components/report-sections/tool-link';
+import { NamedFC } from 'common/react/named-fc';
 
+export type FooterTextProps = { environmentInfoProvider: EnvironmentInfoProvider };
```

Comment:
This is duplicated from 'src/reports/components/report-sections/footer-text.tsx', let's avoid this by defining the type in one place only and importing where we need to use it.

Reply:
Reopen and add a new comment on the updated file.